tests/ttnn/unit_tests/operations/test_moreh_matmul.py

```diff
@@ -167,10 +255,14 @@ def moreh_matmul(params, has_output, compute_kernel_config, device):
         ),  # batched matmul
     ),
 )
+@pytest.mark.parametrize("use_randint", (True, False))
+@pytest.mark.parametrize("dtype", (ttnn.bfloat8_b, ttnn.bfloat16), ids=["bfloat8_b", "bfloat16"])
```

Comment:
Plz consider changing parameter to list instead of using tuple.

Reply:
Changed to lists